collect_app/src/test/java/org/odk/collect/android/widgets/items/SelectOneFromMapWidgetTest.kt

```diff
@@ -0,0 +1,138 @@
+package org.odk.collect.android.widgets.items
+
+import androidx.lifecycle.ViewModel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.hamcrest.Matchers.notNullValue
+import org.javarosa.core.model.SelectChoice
+import org.javarosa.core.model.data.SelectOneData
+import org.javarosa.core.model.data.helper.Selection
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.odk.collect.analytics.Analytics
+import org.odk.collect.android.formentry.FormEntryViewModel
+import org.odk.collect.android.formentry.questions.QuestionDetails
+import org.odk.collect.android.injection.config.AppDependencyModule
+import org.odk.collect.android.support.CollectHelpers
+import org.odk.collect.android.support.MockFormEntryPromptBuilder
+import org.odk.collect.android.support.WidgetTestActivity
+import org.odk.collect.android.widgets.support.QuestionWidgetHelpers.promptWithAnswer
+import org.odk.collect.testshared.RobolectricHelpers.getFragmentByClass
+import org.robolectric.Robolectric
+
+@RunWith(AndroidJUnit4::class)
+class SelectOneFromMapWidgetTest {
+
+    private val activity = Robolectric.setupActivity(WidgetTestActivity::class.java)
```

Comment:
I've got an ongoing discussion (https://github.com/robolectric/robolectric/issues/7041) with the Robolectric team about what we're doing about `setupActivity` deprecation. This is a case where we need it really.